projects/smoke_bts7200/src/main.c

```diff
@@ -0,0 +1,80 @@
+// A simple smoke test for BTS7200
+
+// Periodically take reading from user selected channels and log the result
+// Configurable items: wait time, FRONT or REAR power distro selection, channels to be tested
+#include "bts_7200_current_sense.h"
+#include "current_measurement.h"
+#include "current_measurement_config.h"
```

Comment:
Nit: I'd prefer if you also include `pca9539r_gpio_expander.h` even though it's imported in `bts_7200_current_sense.h` just because it defines symbols used in this file.